clang/lib/Driver/ToolChains/Darwin.h

```diff
@@ -299,6 +299,7 @@ class LLVM_LIBRARY_VISIBILITY Darwin : public MachO {
     TvOS,
     WatchOS,
     DriverKit,
+    XROS,
```

Comment:
`LastDarwinPlatform = DriverKit` needs to be updated to `XROS`.

Reply:
Thanks, will cherry-pick https://github.com/llvm/llvm-project/pull/81011